src/components/Activity/styled.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+const ActivityContainer = styled.section`
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	margin: 0.3rem;
+	border: 1px solid black;
+	border-radius: 15px;
+`;
+
+const DateContainer = styled.section`
+	justify-self: center;
+	margin: 0.25rem 0;
+	h2 {
+		margin: 0;
+	}
+`;
+
+const DistanceContainer = styled.section`
+	margin: 0.75rem 0 0.5rem;
+
+	h3 {
+		margin: 0;
+		font-size: 0.85rem;
+	}
+
+	p {
+		margin: 0.25rem 0 0;
+	}
+`;
+
+const DurationContainer = styled.section`
+	margin: 0.75rem 0 0.5rem;
+
+	h3 {
+		margin: 0;
+		font-size: 0.85rem;
+	}
+
+	p {
+		margin: 0.25rem 0 0;
+	}
+`;
```

Comment:
I think you can write one styled component for `DistanceContainer`, `DurationContainer` and `DateContainer`:

```
const ActivityContainer = styled.section`
	display: grid;
	grid-template-columns: 1fr 1fr 1fr;
	margin: 0.3rem;
	border: 1px solid black;
	border-radius: 15px;

	& :first-child {
		justify-self: center;
		margin: 0.25rem 0;
	}
`;

const Wrapper = styled.section`
	margin: 0.75rem 0 0.5rem;

	h2 {
		margin: 0;
	}

	h3 {
		margin: 0;
		font-size: 0.85rem;
	}

	p {
		margin: 0.25rem 0 0;
	}
`;
```

And every styled component should be saved in a separate file for better understanding. :) 

And one more aspect: I'm not sure if it would be better to work with a list to group date, duration and distance. 🤔 But that is up to you.